src/app/modules/verification/__tests__/verification.controller.new.spec.ts

```diff
@@ -316,4 +331,246 @@ describe('Verification controller', () => {
       })
     })
   })
+
+  describe('handleGetOtp', () => {
+    const MOCK_REQ = expressHandler.mockRequest({
+      body: { fieldId: MOCK_FIELD_ID, answer: MOCK_ANSWER },
+      params: { transactionId: MOCK_TRANSACTION_ID },
+    })
+
+    beforeEach(() => {
+      MockOtpUtils.generateOtpWithHash.mockReturnValueOnce(
```

Comment:
can set default mock value in a beforeAll
```ts
MockOtpUtils.generateOtpWithHash.mockReturnValue(...)
```

then can just call mockReturnValueOnce in the individual tests to override return value a single time if needed

Reply:
will make the changes in #1455